$OpenBSD: patch-ubase_util_ml,v 1.3 2014/09/19 12:56:23 chrisz Exp $

The Uprintf module is broken since OCaml 4.02.
The OCaml stdlib provides a similar function since 3.09.
Use this function instead.

--- ubase/util.ml.orig	Fri Sep 19 12:38:42 2014
+++ ubase/util.ml	Fri Sep 19 12:39:27 2014
@@ -62,7 +62,8 @@ let set_infos s =
   if s <> !infos then begin clear_infos (); infos := s; show_infos () end
 
 let msg f =
-  clear_infos (); Uprintf.eprintf (fun () -> flush stderr; show_infos ()) f
+  clear_infos ();
+  Printf.kfprintf (fun _ -> flush stderr; show_infos ()) stderr f
 
 let msg : ('a, out_channel, unit) format -> 'a = msg
 
